pimpmycause/pimpmycause/templates/layout.html

```diff
@@ -148,7 +148,7 @@ <h4 class="pnl__title">Sign up to our newsletter</h4>
                 <a href="/" class="margin-right-5">
                     <img class="footer__logo" src="{% static 'images/homepage/logo.png' %}"/>
                 </a>
-                <p class="small">Copyright @ {% now "Y" %} PIMP MY CAUSE</p>
+                <p class="small">Copyright @ {% now "Y" %} MarketingKind</p>
```

Comment:
does this need to be `MarketingKind Ltd`?

Reply:
It does not say on the Tasks list

Reply:
The previous copyright also doesn't say Pimp My Cause Limited